XR_APILAYER_NOVENDOR_toolkit/menu.cpp

```diff
@@ -317,14 +317,14 @@ namespace {
                                      0,
                                      (int)ScalingType::MaxValue - 1,
                                      [](int value) {
-                                         std::string labels[] = {"Off", "NIS"};
+                                         std::string labels[] = {"Off", "NIS", "FSR"};
```

Comment:
I think somewhere around L236 needs to be updated to set `m_needRestart` when switching between NIS and FSR, so we display the  "please restart" message.

Reply:
I'll check this out and implement the code logic